src/automanage/azext_automanage/aaz/latest/automanage/configuration_profile/_create.py

```diff
@@ -0,0 +1,264 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "automanage configuration-profile create",
+)
+class Create(AAZCommand):
+    """Create a configuration profile
+
+    :example: create configuration-profile
+        az automanage configuration-profile create -n {profile_name} -g {rg} --configuration {"Antimalware/Enable":false,"Backup/Enable":false,"VMInsights/Enable":true,"AzureSecurityCenter/Enable":true,"UpdateManagement/Enable":true,"ChangeTrackingAndInventory/Enable":true,"GuestConfiguration/Enable":true,"LogAnalytics/Enable":true,"BootDiagnostics/Enable":true}
```

Comment:
The value of `--configuration` is not a valid shorthand syntax expression. We use double quotes to wrap the whole value and use the single quotes to wrap special string in value. I think it should be like this
```
"{'Antimalware/Enable':false,'Backup/Enable':false}"
```

Reply:
It's my bad. Freeform dict doesn't support shorthand syntax. Your example is correct.

Reply:
Agreed for shorthand syntax. This `--configuration` param uses freeformdict which is accepting a json so we can keep this example.